examples/simple_hpc_pbs_peer_network/config.json

```diff
@@ -0,0 +1,118 @@
+{
+    "location": "variables.location",
+    "resource_group": "variables.resource_group",
+    "install_from": "headnode",
+    "admin_user": "hpcadmin",
+    "variables": {
+        "hpc_image": "OpenLogic:CentOS-HPC:7.6:latest",
```

Comment:
Probably should change this to CentOS-HPC:7.7:latest

Reply:
Fixed